common/dependencies.py

```
@@ -0,0 +1,42 @@
+def get_user_repo() -> UserRepository:
+    return UserRepository()
+
+def get_auth_service(
+        repo: Annotated[UserRepository, Depends(get_user_repo)]
+) -> UserService:
+    return UserService(user_repo = repo)
+
+#jwt 인증된 사용자인지 확인
+async def get_current_user(
+        auth: Annotated[HTTPAuthorizationCredentials, Depends(security)],
+        user_service : Annotated[UserService, Depends(get_auth_service)],
```

Ruff (0.14.11)

Unused function argument: `user_service`

(ARG001)